modules/instrument_builder/jsx/react.questions.js

```diff
@@ -37,6 +37,8 @@ class LorisElement extends Component {
       case 'label':
         elementHtml = <p>{element.Description}</p>;
         break;
+      case 'line':
+        elementHtml = <div></div>;
```

Comment:
missing break, so it has a tiny render issue.

Reply:
@kongtiaowang thanks, I added it.